Marlin/src/pins/stm32f4/pins_LERDGE_K.h

```diff
@@ -96,6 +103,13 @@
 //  #define E1_CS_PIN                       PE4
 //#endif
 
+//#define E2_STEP_PIN                       PF4  // best guess
+//#define E2_DIR_PIN                        PF3  // best guess
+//#define E2_ENABLE_PIN                     PF5  // best guess
+//#ifndef E2_CS_PIN
+//  #define E2_CS_PIN                       PB2  // best guess
+//#endif
```

Comment:
Information about this board is hard to find, but these seem like the logical choices for the "extra" motor. If these pins are assigned as "E2" then it can be automatically used as X2, Y2, Z2, Z3, or as an extra mixing stepper, or as an extra motor for a `SINGLENOZZLE`, or whatever….